__tests__/app.test.js

```diff
@@ -111,3 +111,42 @@ describe('GET /api/articles/:article_id', () => {
         });
     });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with all comments for an article', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.comments.length).toBe(2)
+            expect(response.body.comments).toBeSortedBy('created_at', {descending: true})
+            response.body.comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
```

Comment:
You could expect the value of 3 here on line 130 instead of expect.any(Number), which would negate the need for line 132.